css/main.scss

```diff
@@ -198,4 +198,34 @@ pre {
     display: none;
 }
 
+#applications #api-token-fieldest {
+  width: 50%;
+}
+
+#applications #api-token-link {
+  display: block;
+  clear: both;
+}
+
+#frmNewApplication legend {
+  padding: 10px;
+}
+
+#frmNewApplication div {
+    flex: 0.3;
```

Comment:
You can use flex: 1 and flex: 2 or something.
It does not add up to 1, if 1 and 2 will be 1/3 and 2/3 for example.